bluetooth_mesh/interfaces.py

```diff
@@ -257,8 +293,23 @@ async def cancel(self) -> None:
 
     async def attach(
         self, app_defined_root: str, token: int
-    ) -> Tuple[str, Mapping[int, List[Tuple[int, Mapping[str, Any]]]]]:
-        return await self._interface.call_attach(app_defined_root, token)
+    ) -> Tuple[str, Dict[int, Dict[Tuple[int, int], Mapping[str, Any]]]]:
+        path, configuration = await self._interface.call_attach(app_defined_root, token)
+
+        configuration_dict = defaultdict(dict)
+
+        for element_config_list in configuration:
```

Comment:
`for element, element_config in configuration.items()`

Reply:
this is not a dict. this is the list of tuples :/